src/api/app/models/workflow/step/submit_request.rb

```diff
@@ -42,6 +42,9 @@ def submit_package
     bs_request.save!
 
     Workflows::ScmEventSubscriptionCreator.new(token, workflow_run, scm_webhook, bs_request).call
+    payload = scm_webhook.payload
+    payload[:number] = bs_request.number
```

Comment:
No need for this anymore...

Reply:
We still need this, I had to add it back. It is used in the `github/gitlab/gitea_status_reporter.rb:status_options`.